pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+# Description
+
+<!-- Please include a summary of the changes and the related issue. Please also include context and any dependencies required for this change. -->
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+<!-- Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration -->
+
+**Test Configuration**:
+* Firmware version:
+* Hardware:
+* Toolchain:
+* SDK:
+
+Thank you for contributing to the Matierial UI Repository!
```

Comment:
What is the relationship between axios and material UI?
I think the current template is sufficient, but what is the pane this PR is made out of?
Why is it placed directly under the repository?

Reply:
Sorry, I mixed up the pull requests I made for Axios and Material UI, I'll change it.
I just took inspiration from a couple websites that talked about pull request templates and merged them together.